src/player.py

```diff
@@ -2,13 +2,16 @@
 import math
 
 from player_input import PlayerInput
-from weapon import Weapon, Beam, Rocket, MachineGun
+from weapon import Weapon, LaserBeam, Rocket, MachineGun
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
 
 
 class Player:
     # TODO seems like the arcade engine wants us to subclass Sprite for all
     # our game entities.  Seems like composition would be better?
     sprite: arcade.Sprite
+    projectile_list: arcade.SpriteList = None
+    beam_list: arcade.SpriteList = None
```

Comment:
We can remove the `= None` here.  `__init__` immediately initializes these to a non-None value.

Reply:
Fixed